blended-measurement/blended-measurement.tex

```diff
@@ -0,0 +1,685 @@
+each filter.  The most important use case for the Monte Carlo samples [D8] is
+shear estimation for gravitational lensing, but we anticipate it being useful
+for any study of faint objects for which unbiased population statistics are
+more important than precise measurements of individual objects.  The
+optimizer-based fitting is should produce our best astrometric measurements
+for fainter stars, and may yield better galaxy photometry and morphology
+measurements than the deblended coadd fitting.
+
+The simultaneous optimizer-based fit will also be used as templates for
+another round of deblending (as in Section~\ref{sec:models-as-templates}),
+this time producing deblended pixel values for individual visits [D7].  These
+will be used for forced PSF photometry at the per-epoch positions determined
+from the simultaneous multi-epoch fit.  This will populate the forced source
+catalog [D9], which represent our best estimates of the lightcurves of
+faint variable objects.  We use positions from multi-epoch fitting to
+consistently handle stars with significant proper motions, and we perform only
+PSF photometry, as the vast majority of variable objects are indeed point
+sources.
+
+For all multi-epoch measurements, we include models for transient and fast-
+moving objects detected in difference images [D10], as described in
+Section~\ref{sec:time-domain}. These models will have a free flux
+parameter in each epoch, but will have centroids fixed at the position
+determined from detection image(s).
+
+\subsection{Possible Modifications}
+\label{sec:alternatives}
+
+\subsubsection{Likelihood Coadds}
+
+\label{sec:likelihood-coadds}
+
+Likelihood coadds (also known as Kaiser coadds or detection maps) present an
+intriguing but untested alternative to direct coadds.  They represent an
+optimal combination of images in both image quality and S/N (impossible with
+direct coadds), but cannot be interpreted in the same way as traditional
+coadds and single-epoch images, requiring completely new algorithms for all
+operations performed on them.  As a result, making use of likelihood coadds
+may require considerably more human effort, but it could reduce the need for
+multi-epoch processing (but probably not eliminate it). If they prove viable,
+likelihood coadds would replace direct coadds in most or all of the places the
+latter are currently used.
+
+\subsubsection{Model Fluxes on PSF-Matched Coadds}
+
+\label{sec:models-on-psf-matched-coadds}
+
+Forward fitting of galaxy models only formally accounts for differences in PSF
+size across filters when the galaxy model is flexible enough to capture the
+true morphology of the galaxy being fit -- a condition that is never fully
+met in practice.  The best galaxy colors may thus require fitting to
+PSF-matched coadds instead of direct coadds, even though direct coadds may
+allow the model parameters to be constrained better.
+
+If fitting to PSF-matched coadds produces better galaxy colors, we will do
```

Comment:
I think here we have to implement both and compare.  There are no SRD specs in this area to my knowledge, and in fact the only real measures of quality are things whose implementation is outside project scope (e.g. photometric redshift accuracy).  We almost certainly want to collect some code from science collaboration members to be able to do those tests ourselves, of course.